### PR TITLE
Nftest reseqc inferexperiment

### PR DESCRIPTION
**Pull request by pinin4fjords**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/rseqc/inferexperiment/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                    [ id:'test' ], // meta map
+                    file(params.test_data['sarscov2']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true)
+                ]
+    	        input[1] = file(params.test_data['sarscov2']['genome']['test_bed'], checkIfExists: true)
+                """
+            }
+        }
+
+        then {
+            assert process.success
```
Comment (sateeshperi) [inline modules/nf-core/rseqc/inferexperiment/tests/main.nf.test:32]: assertAll()

File: modules/nf-core/rseqc/inferexperiment/tests/main.nf.test [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    tag "rseqc"
+    tag "rseqc/inferexperiment"
+
+    test("sarscov2 - [[meta] - bam] - bed") {
+
+        when {
+            params {
+                // define parameters here. Example:
+                // outdir = "tests/results"
+            }
```
Comment (sateeshperi) [inline modules/nf-core/rseqc/inferexperiment/tests/main.nf.test:19]:
```suggestion
```

File: modules/nf-core/rseqc/inferexperiment/tests/main.nf.test [hunk 3]
```diff
@@ -0,0 +1,38 @@
+nextflow_process {
+
+    name "Test Process RSEQC_INFEREXPERIMENT"
+    script "../main.nf"
+    process "RSEQC_INFEREXPERIMENT"
+    config "./nextflow.config"
+
```
Comment (sateeshperi) [inline modules/nf-core/rseqc/inferexperiment/tests/main.nf.test:7]:
```suggestion
```

Comment (pinin4fjords) [reply modules/nf-core/rseqc/inferexperiment/tests/main.nf.test:7]: Can I keep the spaces? I actually find it much easier to read with things spaced out a bit

File: modules/nf-core/rseqc/inferexperiment/tests/main.nf.test (hunk 1 above)
Comment (pinin4fjords) [reply modules/nf-core/rseqc/inferexperiment/tests/main.nf.test:32]: This is literally what nf-test creates- could that be changed if you don't like this?

File: modules/nf-core/rseqc/inferexperiment/tests/main.nf.test (hunk 3 above)
Comment (maxulysse) [reply modules/nf-core/rseqc/inferexperiment/tests/main.nf.test:7]: I see no issues with keeping an empty line to enhance readability

File: modules/nf-core/rseqc/inferexperiment/tests/main.nf.test (hunk 1 above)
Comment (maxulysse) [reply modules/nf-core/rseqc/inferexperiment/tests/main.nf.test:28]: I'm actually fine with this for now as the test is working.
We should lint for that, but let's merge that one, and do another pass in all modules to be more coherent.
I think I'm starting to have a good global view and some ideas on what we can improve

**Review by maxulysse — APPROVED** [review]

**Pull request merged**